src/coreclr/jit/assertionprop.cpp

```diff
@@ -5260,7 +5260,7 @@ class AssertionPropFlowCallback
     {
         ASSERT_TP pAssertionOut;
 
-        if (predBlock->KindIs(BBJ_COND) && (predBlock->bbJumpDest == block))
+        if (predBlock->KindIs(BBJ_COND) && predBlock->JumpsTo(block))
```

Comment:
At first glace i wasn't certain what `JumpsTo(block)` was doing, it could be getting or setting, readin
g the original made it clear. Consider using `IsJumpTo` which would make it clearer.

Reply:
I would object to `IsJumpsTo()` -- that just doesn't read well. If consensus is that `JumpsTo()` is confusing, I'd recommend leaving these cases as `block->JumpTarget() == block2`

Reply:
I would actually prefer something like `HasJumpTo` since `JumpsTo` to me reads like there's no other option (that is, BBJ_ALWAYS) and the "has" makes it clearer this is a predicate.

Reply:
Either `HasJumpTo` or `JumpTarget() == block2` sound ok to me.